breathecode/authenticate/models.py

```diff
@@ -43,7 +43,13 @@ class Meta:
 class Profile(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     avatar_url = models.CharField(max_length=255, blank=True, null=True, default=None)
-    bio = models.CharField(max_length=255, blank=True, null=True)
+    bio = models.CharField(
+        max_length=255,
+        blank=True,
+        null=True,
+        help_text=
+        'User biography, this will be used the bio in the lang of the user, otherwise frontend will use'
```

Comment:
User biography in user's language. Will be used if there are no ProfileTranslations.